### DISCUSSION TITLE
World are empty

### DISCUSSION THREAD

**Discussion by awesome-ux** [Minestom/Minestom#2286] [Help]
Hey everyone i want to load my lobby, but the problem is the world is always empty

Here is moy code:

WorldInstanceHelper: https://haste.devstorage.eu/lMT1tru4AP
Main: https://haste.devstorage.eu/2pUhpmNa1B

Thanks for helping!

**Comment by UserUNP** [comment]
does `instances/world/` contain a valid minecraft world?

**Comment by ImTheLeviDR** [comment]
I had the same issue. I opened the world in singleplayer and moved the singleplayer world to the world folder. It fixed the problem.
